pkg/components/metadata.go

```diff
@@ -29,6 +29,17 @@ import (
 	"github.com/imdario/mergo"
 )
 
+func generateObjectMeta(name string, ns string, cn string) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Name:        name,
+		Namespace:   ns,
+		ClusterName: cn,
+		Annotations: make(map[string]string),
```

Comment:
whether make(map[string]string, 0) will change anything or both are same Rob?

Reply:
Nope.  No need to give a size when initializing a map, only slice or array.